common/vm_wait_primary_nic_mac.yml

```diff
@@ -41,7 +41,9 @@
     - gather_network_facts.network_data['0'].mac_addr is defined
     - gather_network_facts.network_data['0'].mac_addr
 
-- name: "Display VM's primary network adapter MAC address"
-  ansible.builtin.debug:
-    msg:
-      - "VM primary network adapter MAC address: {{ vm_primary_nic_mac }}"
\ No newline at end of file
+- name: "Assert VM primary network adapter has MAC address"
+  ansible.builtin.assert:
+    that:
+      - vm_primary_nic_mac
```

Comment:
Here need to use 'ansible.utils.hwaddr' to check it?

Reply:
Updated. Thanks!